components/menu/src/index.js

```diff
@@ -15,36 +25,66 @@ const HamburgerButton = (element) => {
     }
   }, [])
 
+  const buttonSize = parseFloat(
+    element.buttonSize || 48
+  ) /* 48 is minimum size for buttons according to Lighthouse */
```

Comment:
This doesn't work? You want it to parse the element.buttonSize attribute, default to 48 and also enforce that it is at least 48. So you need something like this:

```suggestion
  const parsed_button_size = parseFloat( element.buttonSize )
  const button_size = isNaN(parsed_button_size) || parsed_button_size < 48 ? 48 : parsed_button_size
```

Reply:
I don't think we actually want to enforce 48px as a minimum. Just that the default value is based on the recommendation from Lighthouse. But this code could probably be written to make this clearer.

Reply:
I've learned that 48 is a very large minimum. Don't know why lighthouse wants that.

* WGAC 2.5.8, level AA, requires 24x24
* WGAC 2.5.5, level AAA, requires 44x44

I would say we should require 24px and default to 44px.

Reply:
This code has now been simplified, and default changed to 44px.